fuzzers/034b-cmt-mmcm-pips/fixup_and_group.py

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2017-2020  The Project X-Ray Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+"""
+This is a db fixup script that does two things:
```

Comment:
You may remove the bit grouping functionality from this script. It has been incorporated into `dbfixup.py`

Reply:
https://github.com/SymbiFlow/prjxray/pull/1447/files/b81c84d9d0c3648dd46134d379ec4e2677e52b57#diff-66a33010bffb03af1e370edfd5199582R14-R15

The IN_USE stuff is specific to the MMCM and PLL fixup, so it isn't fully encompased by `dbfixup.py`.